lib/gulp-tasks/browsersync.js

```diff
@@ -0,0 +1,70 @@
+/**
+ *  @fileOverview Uses Gulpjs and Browser-Sync to create a server
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:browser-sync
+ *  @requires     ./lib/get-options
+ */
+'use strict';
+
+var browsersync = require('browser-sync'),
+  getOptions = require('../get-options').getOptions;
+
+/**
+ * Function to get default options for an implementation of browser-sync
+ * @function
+ * @name getDefaultOptions
+ * 
+ * @returns {Object}  options  an object of default browser-sync options
+ */
+function getDefaultOptions () {
+
+  /* default options for pattern-importer */
+  var options = {
+
+    environment: {
+      config: {
+        server: {
+          baseDir: 'app/'
```

Comment:
`./` is fine, or `app` if it is documented somewhere. `app/` looks just too related to our system on my opinion